package.json

```diff
@@ -72,7 +72,7 @@
     "react": ">=0.14.1"
   },
   "engines": {
-    "node": "^10.13.0 || ^12.x",
+    "node": "^10.13.0 || >= 12.x",
```

Comment:
what would the difference be with:  `>= 10.x` or event being very explicit `10.x || 11.x || 12.x || 13.x`.

Reply:
With `>= 10.x` will match Node.js version 10 and up. With `10.x || 11.x || 12.x || 13.x` does the same as `>= 10.x` but will not match upcoming versions (Node.js 14 is around the corner).

Reply:
I can update it to `>= 10.x` and the tests pipeline accordingly.

Reply:
I pushed a commit with the `node` engine property updated to `>= 10.x`.